project/BuildTarget.scala

```diff
@@ -205,3 +220,75 @@ case class DebBuildTarget(distributions: Seq[String], components: String, depend
     AdditionalIO.setExecutable(stage / "command")
   }
 }
+
+case class TarGzSelfContainedExecutableBuildTarget(libs: Seq[String]) extends BuildTarget {
+  val architecture = "amd64"
+
+  override def launcherExecHook: String = "exec $DIR/lib/ld-musl-x86_64.so.1"
```

Comment:
The dynamic linker, `ld-musl-x86_64.so.1`, is also a static binary so we can simply exec it followed by our binary (`rp`), without having to change the hard-coded path to the linker within the binary itself.

Reply:
Neat!